### PR TITLE
Add water velocity.

### PR DESCRIPTION
**Pull request by arjo129**
This micro-PR adds water velocity as one of the parameters we care about.

Signed-off-by: Arjo Chakravarty <arjo@openrobotics.org>

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
I'm also populating this field in #81 . My concern with the `ROSToFSK` function is that it's just flipping axes, but not taking the whole vehicle pose into account.

If we want to get something in quick just to make sure the field is populated, I'm ok with merging this PR, but heads up that the values being published here aren't correct.

File: lrauv_ignition_plugins/src/TethysCommPlugin.cc [hunk 1]
```diff
@@ -586,7 +586,10 @@ void TethysCommPlugin::PostUpdate(
 
   // Water velocity
   // rateUVW
-  // TODO(anyone)
+  // TODO(arjo): include currents in water velocity?
+  auto localVel = modelPose.Rot().Inverse() * veloGround;
+  ROSToFSK(localVel);
```
Comment (tfoote) [inline lrauv_ignition_plugins/src/TethysCommPlugin.cc:591]:
```suggestion
  //TODO(louise) check for translation/position effects
  ROSToFSK(localVel);
```

Comment (tfoote) [reply lrauv_ignition_plugins/src/TethysCommPlugin.cc:591]: Add a comment to track Louise's question.

**Review by tfoote — APPROVED** [review]
add one more comment/todo to track louise's concern.

**Pull request merged**